hw/top_darjeeling/sw/autogen/top_darjeeling_soc_mbx.h

```diff
@@ -0,0 +1,431 @@
+ * memory-mapped registers associated with this peripheral should have an
+ * address between #TOP_DARJEELING_SOC_MBX_MBX5_SOC_BASE_ADDR and
+ * `TOP_DARJEELING_SOC_MBX_MBX5_SOC_BASE_ADDR + TOP_DARJEELING_SOC_MBX_MBX5_SOC_SIZE_BYTES`.
+ */
+#define TOP_DARJEELING_SOC_MBX_MBX5_SOC_SIZE_BYTES 0x20u
+
+/**
+ * Peripheral base address for soc device on mbx6 in top darjeeling.
+ *
+ * This should be used with #mmio_region_from_addr to access the memory-mapped
+ * registers associated with the peripheral (usually via a DIF).
+ */
+#define TOP_DARJEELING_SOC_MBX_MBX6_SOC_BASE_ADDR 0x1465600u
+
+/**
+ * Peripheral size for soc device on mbx6 in top darjeeling.
+ *
+ * This is the size (in bytes) of the peripheral's reserved memory area. All
+ * memory-mapped registers associated with this peripheral should have an
+ * address between #TOP_DARJEELING_SOC_MBX_MBX6_SOC_BASE_ADDR and
+ * `TOP_DARJEELING_SOC_MBX_MBX6_SOC_BASE_ADDR + TOP_DARJEELING_SOC_MBX_MBX6_SOC_SIZE_BYTES`.
+ */
+#define TOP_DARJEELING_SOC_MBX_MBX6_SOC_SIZE_BYTES 0x20u
+
+/**
+ * Peripheral base address for soc device on mbx_pcie0 in top darjeeling.
+ *
+ * This should be used with #mmio_region_from_addr to access the memory-mapped
+ * registers associated with the peripheral (usually via a DIF).
+ */
+#define TOP_DARJEELING_SOC_MBX_MBX_PCIE0_SOC_BASE_ADDR 0x1460100u
+
+/**
+ * Peripheral size for soc device on mbx_pcie0 in top darjeeling.
+ *
+ * This is the size (in bytes) of the peripheral's reserved memory area. All
+ * memory-mapped registers associated with this peripheral should have an
+ * address between #TOP_DARJEELING_SOC_MBX_MBX_PCIE0_SOC_BASE_ADDR and
+ * `TOP_DARJEELING_SOC_MBX_MBX_PCIE0_SOC_BASE_ADDR + TOP_DARJEELING_SOC_MBX_MBX_PCIE0_SOC_SIZE_BYTES`.
+ */
+#define TOP_DARJEELING_SOC_MBX_MBX_PCIE0_SOC_SIZE_BYTES 0x20u
+
+/**
+ * Peripheral base address for soc device on mbx_pcie1 in top darjeeling.
+ *
+ * This should be used with #mmio_region_from_addr to access the memory-mapped
+ * registers associated with the peripheral (usually via a DIF).
+ */
+#define TOP_DARJEELING_SOC_MBX_MBX_PCIE1_SOC_BASE_ADDR 0x1460200u
+
+/**
+ * Peripheral size for soc device on mbx_pcie1 in top darjeeling.
+ *
+ * This is the size (in bytes) of the peripheral's reserved memory area. All
+ * memory-mapped registers associated with this peripheral should have an
+ * address between #TOP_DARJEELING_SOC_MBX_MBX_PCIE1_SOC_BASE_ADDR and
+ * `TOP_DARJEELING_SOC_MBX_MBX_PCIE1_SOC_BASE_ADDR + TOP_DARJEELING_SOC_MBX_MBX_PCIE1_SOC_SIZE_BYTES`.
+ */
+#define TOP_DARJEELING_SOC_MBX_MBX_PCIE1_SOC_SIZE_BYTES 0x20u
+
+
+
+#define PINMUX_MIO_PERIPH_INSEL_IDX_OFFSET 2
+
+// PERIPH_INSEL ranges from 0 to NUM_MIO_PADS + 2 -1}
+//  0 and 1 are tied to value 0 and 1
+#define NUM_MIO_PADS 12
+#define NUM_DIO_PADS 73
+
+#define PINMUX_PERIPH_OUTSEL_IDX_OFFSET 3
+
+/**
+ * Pinmux Peripheral Input.
+ */
+typedef enum top_darjeeling_pinmux_peripheral_in {
```

Comment:
Also here - probably because the same helper does not detect there is no `pinmux` and other IPs down below.